mojo/services/view_manager/public/cpp/BUILD.gn

```diff
@@ -43,9 +43,8 @@ mojo_sdk_source_set("cpp") {
   ]
 
   mojo_sdk_deps = [
-    "mojo/public/c/gles2:headers",
```

Comment:
This matches https://github.com/domokit/mojo/commit/e2fb76e3f873dbd86d685fa8823bfbd427fc477c.  I'll do a roll of everything once I've deleted more dependencies.